popups.py

```diff
@@ -0,0 +1,44 @@
+import folium
+import pandas as pd
+import streamlit as st
+from streamlit_folium import st_folium
+from folium.plugins import MarkerCluster
+import math
+
+# these are the CSV files of the capitals and countries, both located in data
```

Comment:
do you mean states and counties?

Reply:
Yeah countries was a typo, it should be counties. The other csv contains geographical information for state capitals, which we are using to place the markers for the states themselves